src/features/device-state/middleware.ts

```diff
@@ -35,6 +35,7 @@ export const checkDeviceExistsIsFrozen = multiCacheMemoizee(
 		undefinedAs: false,
 		maxAge: DEVICE_EXISTS_CACHE_TIMEOUT,
 	},
+	{ useVersion: false },
```

Comment:
A comment above the query that in case we change the query we need to change the `useVersion` to `true` would be nice.

Reply:
I've added an explicit return type to the memoized function so that if the query changes without updating the memoized fn then typescript will raise it